code/addons/onboarding/src/features/SplashScreen/SplashScreen.tsx

```diff
@@ -178,6 +208,36 @@ export const SplashScreen = ({ onDismiss }: SplashScreenProps) => {
       <Backdrop />
       <Content visible={visible}>
         <h1>Meet your new frontend workshop</h1>
+        <Features>
+          <div>
+            <svg xmlns="http://www.w3.org/2000/svg" width="33" height="32">
```

Comment:
would be nice to separate them in different svg files. Or even better, to have them come from @storybook/icons instead

Reply:
These are one-off icons, not used anywhere else. That's intentional.